Events/index.md

```diff
@@ -48,5 +48,9 @@ The First Annual [Walk for T1D: For a world without type 1 diabetes](http://www2
   In this episode we hear from [Todd Palmer](http://ne.oregonstate.edu/todd-s-palmer), professor of nuclear engineering, about his use of Monte Carlo simulations in nuclear fuel research. In addition, find out how [Haizhong Wang](http://cce.oregonstate.edu/wang), assistant professor in transportation engineering, is incorporating Monte Carlo to model tsunami evacuation routes in the Cascadia Subduction Zone.
 
 # Upcoming Defenses
+* [Doug Woods](https://rtrp.github.io/osu-transport/users/woodsdou/)
```

Comment:
ohhhhh daaaaang

👍

Reply:
Hehe, I thought you'd like that!

Reply:
Oh man...you are gonna beat Trevor :D

Reply:
You probably are! But Jordan is going to beat everyone... @dougwoods3 has TP already edited your dissertation??

Reply:
Oh no, TP hasn't seen it yet because I'm still writing it.

When does Jordan defend?

Reply:
I don't think he has a date yet but his he says his dissertation has been reviewed and he has been given the green-light...